designer/client/package.json

```diff
@@ -68,7 +69,7 @@
         "prop-types": "15.8.1",
         "query-string": "7.1.0",
         "react": "17.0.2",
-        "react-ace": "8.0.0",
+        "react-ace": "10.1.0",
```

Comment:
Maybe add some "TODO: review patches when upgrading this dependency"?

Reply:
In json? :)